### ISSUE TITLE
When I say some key words ('Time' 'Weather' 'Life' 'Joke') ,there is no low beeps or response back from Jasper. Can you help me ?

### ISSUE THREAD

**Issue by Liujiming** [jasperproject/jasper-client#573]
----debug info ---
i@raspberrypi ~/jasper $ ./jasper.py --debug
DEBUG:client.tts:Saying 'How can I be of service, yan?' with 'espeak-tts'
DEBUG:client.tts:Executing espeak -v default+m3 -p 40 -s 160 -w /tmp/tmpYkWH0H.wav 'How can I be of service, yan?'
DEBUG:client.tts:Executing aplay -D hw:1,0 /tmp/tmpYkWH0H.wav
DEBUG:client.tts:Output was: 'Playing WAVE '/tmp/tmpYkWH0H.wav' : Signed 16 bit Little Endian, Rate 22050 Hz, Mono
'
DEBUG:client.brain:Looking for modules in: '/home/pi/jasper/client/modules'
DEBUG:client.brain:Found module 'Birthday' with words: ['BIRTHDAY']
DEBUG:client.brain:Found module 'Gmail' with words: ['EMAIL', 'INBOX']
DEBUG:client.brain:Found module 'HN' with words: ['HACKER', 'NEWS', 'YES', 'NO', 'FIRST', 'SECOND', 'THIRD']
DEBUG:client.brain:Found module 'Joke' with words: ['JOKE', 'KNOCK KNOCK']
DEBUG:client.brain:Found module 'Life' with words: ['MEANING', 'OF', 'LIFE']
DEBUG:client.brain:Found module 'MPDControl' with words: ['MUSIC', 'SPOTIFY']
DEBUG:client.brain:Found module 'News' with words: ['NEWS', 'YES', 'NO', 'FIRST', 'SECOND', 'THIRD']
DEBUG:client.brain:Found module 'Notifications' with words: ['FACEBOOK', 'NOTIFICATION']
DEBUG:client.brain:Found module 'Random' with words: ['FIST BUMP']
DEBUG:client.brain:Found module 'Time' with words: ['TIME']
DEBUG:client.brain:Found module 'Unclear' with words: []
DEBUG:client.brain:Found module 'Weather' with words: ['WEATHER', 'TODAY', 'TOMORROW']
INFO:apscheduler.scheduler:Scheduler started
DEBUG:apscheduler.scheduler:Looking for jobs to run
DEBUG:apscheduler.scheduler:No jobs; waiting until a job is added
INFO:apscheduler.scheduler:Added job "Notifier.gather" to job store "default"
INFO:client.conversation:Starting to handle conversation with keyword 'JASPER'.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:apscheduler.scheduler:Looking for jobs to run
DEBUG:apscheduler.scheduler:Next wakeup is due at 2016-10-23 23:00:21.320422+00:00 (in 29.986689 seconds)
INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): www.google.com
/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmou**********YacEp_PpugTAaN4&output=json HTTP/1.1" 200 None
INFO:client.stt:Transcribed: (u'PATH', u'PAN', u'PAM', u'CAB', u'CAMP')
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:apscheduler.scheduler:Looking for jobs to run
INFO:apscheduler.executors.default:Running job "Notifier.gather (trigger: interval[0:00:30], next run at: 2016-10-23 23:00:21 UTC)" (scheduled at 2016-10-23 23:00:21.320422+00:00)
DEBUG:apscheduler.scheduler:Next wakeup is due at 2016-10-23 23:00:51.320422+00:00 (in 29.997684 seconds)
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:apscheduler.scheduler:Looking for jobs to run
WARNING:apscheduler.scheduler:Execution of job "Notifier.gather (trigger: interval[0:00:30], next run at: 2016-10-23 23:00:51 UTC)" skipped: maximum number of running instances reached (1)
DEBUG:apscheduler.scheduler:Next wakeup is due at 2016-10-23 23:01:21.320422+00:00 (in 29.993517 seconds)
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmou**********YacEp_PpugTAaN4&output=json HTTP/1.1" 200 None
WARNING:client.stt:Empty response: Nothing has been transcribed.
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:apscheduler.scheduler:Looking for jobs to run
WARNING:apscheduler.scheduler:Execution of job "Notifier.gather (trigger: interval[0:00:30], next run at: 2016-10-23 23:01:21 UTC)" skipped: maximum number of running instances reached (1)
DEBUG:apscheduler.scheduler:Next wakeup is due at 2016-10-23 23:01:51.320422+00:00 (in 29.996280 seconds)
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmou**********YacEp_PpugTAaN4&output=json HTTP/1.1" 200 None
INFO:client.stt:Transcribed: (u'PAM', u'CAMP', u'PENN', u'HAM', u'PEN')
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmou**********YacEp_PpugTAaN4&output=json HTTP/1.1" 200 None
INFO:client.stt:Transcribed: (u'TIME', u'HAM', u'TYPE', u'HI')
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmou**********YacEp_PpugTAaN4&output=json HTTP/1.1" 200 None
WARNING:client.stt:Empty response: Nothing has been transcribed.
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
No disturbance detected
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'
DEBUG:apscheduler.scheduler:Looking for jobs to run
WARNING:apscheduler.scheduler:Execution of job "Notifier.gather (trigger: interval[0:00:30], next run at: 2016-10-23 23:01:51 UTC)" skipped: maximum number of running instances reached (1)
DEBUG:apscheduler.scheduler:Next wakeup is due at 2016-10-23 23:02:21.320422+00:00 (in 29.996762 seconds)
No disturbance detected

**Comment by G10DRAS** [comment]
You are using GoogleSTT for both active and passive listening.
Seems keyword 'JASPER' never transcribed by GoogleSTT and recognized by Jasper in your case, that's why you didn't heard any beep after when you say 'JASPER'.
Try changing keyword JASPER in jasper.py to any other word that better suits your ascent.

Did you heard 'How can I be of service, yan?'  in speaker when you started Jasper ?

**Comment by Liujiming** [comment]
@G10DRAS 
Yes.I heard 'How can I be of service, yan?' in speaker when I started Jasper .
When I say 'JASPER' ,it gives me a high beep.
But when I say 'TIME' or 'What's the time now?' ,it gives me nothing.(Now low beep or any voice)It seems not to be recognized by Jasper.

--debug--
DEBUG:requests.packages.urllib3.connectionpool:"POST /speech-api/v2/recognize?lang=en-us&pfilter=2&maxresults=6&client=chromium&key=AIzaSyCMZwdmouYacEp_PpugTAaN4&output=json HTTP/1.1" 200  
INFO:client.stt:Transcribed: (u'TIME', u'HAM', u'TYPE', u'HI')
DEBUG:client.conversation:Stopped listening for keyword 'JASPER'
INFO:client.conversation:Nothing has been said or transcribed.
DEBUG:client.conversation:Started listening for keyword 'JASPER'

**Comment by G10DRAS** [comment]
Flow is like:
1. You say keyword "JASPER"
2. Jasper recognized keyword and respond with a HIGH BEEP
3. Then you say command "Whats the time now"
4. Jasper record it and respond with LO BEEP
5. Jasper answer "it is blah blah right now"

Did you follow the same ?

**Comment by G10DRAS** [comment]
or see this  [issue](https://github.com/jasperproject/jasper-client/issues/563) it looks similar.

**Comment by Liujiming** [comment]
@G10DRAS 
Flow is like:
1. I say keyword "JASPER"
2. Jasper recognized keyword and respond with a HIGH BEEP
3. Then I say command "Whats the time now"
4. Jasper just gave me what I said (like this ,INFO:client.stt:Transcribed: (u'Whats the time now'))  and respond with no BEEP or sound.
5. Jasper no answer me anything.

**Comment by G10DRAS** [comment]
after INFO:client.stt:Transcribed: (u'Whats the time now')) message, 
what is there in debug log ? 
or try help yourself by digging in to mic.py code by putting some debug log messages.

**Comment by Liujiming** [comment]
@G10DRAS 
OK.I will follow what you say. Thank you very much.

**Comment by diegodeodati** [comment]
I have the same issue after the first beep i say time and nothing happen after second low beep...

**Comment by G10DRAS** [comment]
Did you check debug log?

**Comment by diegodeodati** [comment]
Yes, i will solve it throw modify mic.py and introduce an different loop timing in active listener and also remove the break (i have to consider it)

i would to add:

for i in range (0,100):

time.sleep(0.5)

something like this

**Comment by raghuramtce** [comment]
When i started jasper i am nt able to hear 'How can I be of service, yan?' in speaker.. can anybody help?
